virtwho/register.py

```diff
@@ -18,9 +18,10 @@ def __init__(self, host, username, password, port=22, register_type='rhsm',
         :param username: account username of the host
         :param password: password to access the host
         :param port: port to access the host
-        :param register_type: rhsm or satellite, rhsm as default
+        :param register_type: rhsm/rhsm_product/satellite, rhsm as default
```

Comment:
In my opinion, there is no need to define rhsm_product account, as users, you can config rhsm for stage or product. it's not a good idea to manage or use two or more rhsm accounts at the same time.